tests/test_fetcher.py

```diff
@@ -223,47 +223,13 @@ def initialize(context):
     context.stocks = [sid(3766), sid(25317)]
 
 def handle_data(context, data):
-    record(ibm_signal=data[sid(3766)]["signal"])
-    record(dell_signal=data[sid(25317)]["signal"])
-
-    assert "signal" not in data[sid(24)]
+    record(ibm_signal=data.current(sid(3766), "signal"))
+    record(dell_signal=data.current(sid(25317), "signal"))
     """)
 
         self.assertEqual(5, results["ibm_signal"].iloc[-1])
         self.assertEqual(5, results["dell_signal"].iloc[-1])
 
-    def test_fetch_csv_with_nomatch_symbol(self):
```

Comment:
I killed this test because with our ffilling behaviour, the test isn't accurate anymore.